test/templateCount.test.ts

```diff
@@ -21,7 +21,7 @@ function addSuite(source: TemplateSource | undefined): void {
             [ProjectLanguage.JavaScript, FuncVersion.v3, 14],
             [ProjectLanguage.CSharp, FuncVersion.v1, 12],
```

Comment:
Would you not have to change the count of v1 triggers too?

Reply:
It was already in there for v1